code/lab/manager/lab_manager.cpp

```diff
@@ -396,6 +577,7 @@ void LabManager::changeDisplayedObject(LabMode mode, int info_index, int subtype
 	//if (mode == CurrentMode && info_index == CurrentClass)
 		//return;
 
+	// THIS IS BUGGY FIXMEE
```

Comment:
OH NO!

Reply:
Ha! That's why I put comments in.. so I don't forget to fix this. It worked.

Reply:
Fixed